src/corelibs/U2Formats/src/PDBFormat.cpp

```diff
@@ -426,7 +426,11 @@ void PDBFormat::PDBParser::parseAtom(BioStruct3D &biostruct, U2OpStatus &) {
     biostruct.modelMap[currentModelIndex + 1].insert(id, a);
 
     if (atomIsInChain) {
+        SAFE_POINT_EXT(0 <= chainIndex && chainIndex < biostruct.moleculeMap.size(), os.setError("Unexpected chainIndex"), );
```

Comment:
The error is expected to be shown to a user and chainIndex/currentModelIndex is not known to the user.

Reply:
You mean, that this value should be added to the message?